src/overlays/misc/ovl_kaleido_scope/z_kaleido_map.c

```diff
@@ -764,25 +765,25 @@ void KaleidoScope_DrawWorldMap(PlayState* play, GraphicsContext* gfxCtx) {
 
         // Quad PAGE_BG_QUADS + WORLD_MAP_QUAD_28
 
-        pauseCtx->mapPageVtx[172].v.ob[0] = pauseCtx->mapPageVtx[174].v.ob[0] =
+        pauseCtx->mapPageVtx[43 * 4 + 0].v.ob[0] = pauseCtx->mapPageVtx[43 * 4 + 2].v.ob[0] =
```

Comment:
use `PAGE_BG_QUADS + WORLD_MAP_QUAD_28` instead of `43` here?

Reply:
I didn't because of how much clutter it would add, you think it's fine?

Reply:
it's kinda weird that everything else is technically shiftable if you add a new enum (I hope), but not this one. Does it really look so bad? Could also give a name to `PAGE_BG_QUADS + WORLD_MAP_QUAD_28` instead

Reply:
would be fine with either fwiw, as long as there is a good name for a wrapper macro

Reply:
43 -> `PAGE_BG_QUADS + WORLD_MAP_QUAD_28` 9c4bcb93ec39af30cda1d638d25be2d136f98d5c